strongbox-commons/src/main/java/org/carlspring/strongbox/booters/TempDirBooter.java

```diff
@@ -1,19 +1,18 @@
 package org.carlspring.strongbox.booters;
 
+import javax.annotation.PostConstruct;
```

Comment:
I'm using the Strongbox supplied IDEA formatters. This was an optimize imports command that re-ordered. Will see what Sonar has to say about it.